R/create_files.R

```diff
@@ -1,4 +1,4 @@
-#' Create files
+#' #' Create files
```

Comment:
```suggestion
#' Create files
```
This does not need to have a double `#'` right?